e3db/tests/test_integration.py

```diff
@@ -859,6 +874,7 @@ def test_read_write_config(self):
         """
 
         config_client_public_key, config_client_private_key = e3db.Client.generate_keypair()
+        config_client_public_signing_key, config_client_private_signing_key = e3db.Client.generate_signing_keypair()
```

Comment:
We still need to ensure that clients without singing keys can be successfully read from disk. This may require a new test to be added. In a similar vein to the test for no client email.

Reply:
I added a new test to ensure clients without signing keys work (pretty much the same as the no client email test). I also made sure to run the tests without signing keys for the clients in the `setup_class` method. All tests pass except `test_read_write_config` because it tries to overwrite my `~/.tozny/e3db.json`, so it fails there by design. 

There are some additional changes to ensure backwards compatibility, too: Signing keys for the Config object are now optional parameters (set to empty strings by default) and Client initialization checks for version number before creating `TSV1E3dbAuth`. I am pretty certain that there should not be issues with that, and it seems necessary to include this check because its init will fail if its a v1 client without signing keys. 

Other than that, in the test for no client email I added the `version` field to its config now that it is being checked in `Client`.

Reply:
Ah, will add a test specifically for loading config from disk.

Reply:
Added a test that pretty much mirrors `test_read_write_config` except that the config that is written and subsequently read does not include signing keys. 

All tests pass. `test_read_write_config` and `test_read_config_no_signing_keys` will pass as long as `~/.tozny/` does not already have an `e3db.json`. `test_read_write_config` was originally written to throw an exception on purpose if that is the case.

Reply:
When you say doesn't have signing keys, does that mean that the key (as in key value pair) is not present or that the value associated with the key is an empty string?